development/kyma-github-connector/githubWebhookGateway/pkg/httperrors/httperrors.go

```diff
@@ -0,0 +1,61 @@
+package httperrors
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/kyma-project/test-infra/development/kyma-github-connector/githubWebhookGateway/pkg/apperrors"
+	log "github.com/sirupsen/logrus"
+)
+
+type ErrorResponse struct {
+	Code  int    `json:"code"`
+	Error string `json:"error"`
+}
+
+func AppErrorToResponse(appError apperrors.AppError) (int, ErrorResponse) {
+	httpCode := errorCodeToHttpStatus(appError.Code())
+	errorMessage := appError.Error()
+	return httpCode, ErrorResponse{httpCode, fmt.Sprintf("%s: %s", appError.Desc(), errorMessage)}
+}
+
+func errorCodeToHttpStatus(code int) int {
+	switch code {
+	case apperrors.InternalError:
+		return http.StatusInternalServerError
```

Comment:
If you set the default returned value as `http.StatusInternalServerError` then this case seems to be useless. There would be no real benefit in time execution. We can assume by default that `InternalError` is returned by default, which you've done.

Reply:
You right. Will fix that in separate PR.